code/modules/clothing/under/clerksuits.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "A tailored black suit, used by company security agents."
 	icon = 'icons/obj/clothing/ego_gear/lcuniform.dmi'
 	worn_icon = 'icons/mob/clothing/ego_gear/under.dmi'
-	icon_state = "agent"
+	icon_state = "Templar Knight"
```

Comment:
would be nice if we kept agent here and in the j-corp uniform for linter reasons

Reply:
Fucked up